pkg/volume/fc/fc_util_test.go

```diff
@@ -103,7 +103,7 @@ func TestSearchDisk(t *testing.T) {
 		{
 			name: "PCI disk",
 			wwns: []string{"500a0981891b8dc5"},
-			lun:  "0",
+			lun:  "10",
```

Comment:
Instead of changing from "0" to "10", can you keep the test with lun "0" and add a new test with lun "10"?